src/trie/trie_type.c

```diff
@@ -256,11 +281,12 @@ void TrieType_AofRewrite(RedisModuleIO *aof, RedisModuleString *key, void *value
     rune *rstr;
     t_len len;
     float score;
+    RSPayload payload = {.data = NULL, .len = 0};
 
-    while (TrieIterator_Next(it, &rstr, &len, &score, NULL)) {
-      size_t slen;
+    while (TrieIterator_Next(it, &rstr, &len, &payload, &score, NULL)) {
+      size_t slen = 0;
       char *s = runesToStr(rstr, len, &slen);
-      RedisModule_EmitAOF(aof, TRIE_ADD_CMD, "sbd", key, s, slen, (double)score);
+      RedisModule_EmitAOF(aof, TRIE_ADD_CMD, "sbdbb", key, s, slen, (double)score, "PAYLOAD", 7, payload.data, payload.len);
```

Comment:
This will cause the trie to be filled with empty payloads after reloads. I will rewrite it so that that it will not emit payloads if the payload is empty.